Makefile

```diff
@@ -967,6 +967,8 @@ ifeq ($(filter -DROCKSDB_LITE,$(OPT)),)
 	sh tools/rocksdb_dump_test.sh
 endif
 endif
+	$(MAKE) check-format
+	$(MAKE) check-buck-targets
```

Comment:
They are not included in PR summary. Is it intended? I have concerns on running build_tools/format-diff.sh in simple "make check". build_tools/format-diff.sh has dependencies of clang-format, but I'm not sure how widely this is available. It can also be hard to fetch in some environment. I'm not sure but I've never checked FreeBSD, OpenBSD or Cygwin. Yet "make check" is a simply a place to run all tests, even if users don't plan to contribute any code. This dependency is a little bit too much for me.

Reply:
Yes these two lines are intended, per prior discussion.
I do not have strong opinion with them and are totally OK to remove them, given that we already can run `make check-format` and `make check-buck-targets` separately.

Reply:
We should have a Make target that is a nominal set of checks to run before submitting a PR, and these checks are high value (catching things that should be fixed for low cost) so belong in such a check.

If you want that to be something other than "make check," OK, but don't throw away having a simple way to run a nominal set of checks.